src/data/markdown/docs/20 jslib/01 jslib/01 aws/S3Client/00 copyObject.md

```diff
@@ -0,0 +1,49 @@
+---
+title: 'S3Client.copyObject'
+description: 'S3Client.copyBucket copies an object from a bucket to another'
+excerpt: 'S3Client.copyBucket copies an object from a bucket to another'
+---
+
+`S3Client.copyBucket` copies an object from one bucket to another.
+
+| Parameter            | Type   | Description                                        |
+| :------------------- | :----- | :------------------------------------------------- |
+| sourceBucket         | string | Name of the bucket to copy the object from.        |
+| sourceObjectKey      | string | Name of the object to copy from the source bucket. |
+| destinationBucket    | string | Name of the bucket to copy the object to.          |
+| destinationObjectKey | string | Name of the object to copy from the source bucket. |
```

Comment:
Another question, for the `destinationObjectKey` parameter, does that have to match the `sourceObjectKey` value? Or can the user just set it to the name of the new object? If it's the latter, I think we could change the description for `destinationObjectKey`.

Reply:
The latter. Judging by the methods REST API endpoint documentation, there is no requirement for the source object key and the destination object key to match, I will update the descriptions accordingly 👍🏻